### ISSUE TITLE
Getting same completions with different prefixes

### ISSUE THREAD

**Issue by jojojames** [nathankot/company-sourcekit#20]
Getting same offsets (and same completions) while adding extra characters to the prefix.

Added a dropbox video.

https://dl.dropboxusercontent.com/u/11400324/Untitled.mov

import UIKit

class SentBottlesController: UIViewController {

```
override func loadView() {
    self.view = SentBottlesView.init()
    self.[view] <-- every letter in view returns the same completions (incorrect candidates)
}

override func viewDidLoad() {
    super.viewDidLoad()
    self.title = "Sent"
}
```

}

**Comment by nathankot** [comment]
@jojojames sorry for the slow turnaround on this, this should now be resolved by 5ab8e95 :)

**Comment by jojojames** [comment]
Thanks!
